### ISSUE TITLE
Does oTranscribe transcribe the audio for you?

### ISSUE THREAD

**Issue by effectively0** [oTranscribe/oTranscribe#107]
I feel a bit silly asking, but does this software actually transcribe the audio for you?
I thought it analysed speech and produced transcript text output from audio file input, but it the web app doesnt seem to do anything as it plays back audio.

**Comment by feltoar** [comment]
> **Does oTranscribe automatically convert audio into text?**
> No, it doesn’t. oTranscribe is designed to make the manual task of transcribing audio a little less painful.

https://otranscribe.com/help/#does_otranscribe_automatically_convert_audio_into_text

**Comment by Leftium** [comment]
I think oTranscribe could "automatically transcribe" YouTube videos by importing automatically generated subtitle files. I'm not sure if the oTranscribe webapp can access YouTube subtitle files via the API or something...

What I do is [download the TTML format subtitles using youtube-dl](https://github.com/ytdl-org/youtube-dl/blob/master/README.md#subtitle-options). If you are the owner of the video, you can also access the SBV format subtitle files from YouTube Studio.

Then I wrote a CLI tool that converts TTML/SBV format files to OTR format. If there is interest, I can publish the CLI tool on NPM.

If you need to transcribe a non-YouTube file, you can upload the file to YouTube, then download the automatically generated subtitles.

---

Below is a screen capture and sample OTR file export for this video: http://otranscribe.com/?v=fCn8zs912OE

Try importing this OTR file yourself: [The Zipf Mystery-fCn8zs912OE.en.otr.txt](https://github.com/oTranscribe/oTranscribe/files/4497989/The.Zipf.Mystery-fCn8zs912OE.en.otr.txt)


![otranscribe](https://user-images.githubusercontent.com/381217/79673354-a158f900-819e-11ea-8d00-2ab9d52f0e0c.png)

**Comment by candideu** [comment]
Check out VOSK browser: https://github.com/ccoreilly/vosk-browser.

Their demo shows how you can either upload a file or dictate: https://ccoreilly.github.io/vosk-browser/

Would be great if those two apps could be merged together.

**Comment by Rich2020** [comment]
Hmm, the name is totally misleading. It does no transcription whatsoever.

**Comment by gullabi** [comment]
Concerning this issue, and as @candideu suggested we have developed our own fork to implement this functionality, and it can be found [here](https://github.com/projecte-aina/oTranscribe-plus).

In a nutshell, we have integrated the vosk-browser functionality in oTranscribe and additionally added automated timestamp feature. It can transcribe any file introduced from the file system with timestamps put for each minute. Since vosk-browser works offline, no file is communicated with outside and everything is done with the resources of the local machine. 

We keep our repository as a fork since our intention is to introduce the changes to the original repo, if the maintainer is interested. 

Finally, we will provide with a publicly available deployed version of the app and a desktop version soon. But in the meantime we appreciate any help, QA or suggestions from the community.
